src/library/lib.js

```diff
@@ -148,6 +148,7 @@ filepicker.extend(function(){
         options.storeLocation = store_options.location || 'S3';
         options.storePath = store_options.path;
         options.storeContainer = store_options.storeContainer || store_options.container;
+        options.storeRegion = store_options.storeRegion || store_options.region;
```

Comment:
Okay. But maybe in this case it has more sense to use storeRegion, because we have storeContainer and this two options usually goes together.